src/Data/SparseData.cpp

```diff
@@ -37,43 +37,40 @@
 #include <boost/fusion/adapted/std_pair.hpp>
 #include <shark/Data/SparseData.h>
 
+using namespace shark;
+
 namespace {
 
-typedef std::pair<int, std::vector<std::pair<std::size_t, double> > > LibSVMPoint;
+typedef std::pair<double, std::vector<std::pair<std::size_t, double> > > LibSVMPoint;
 inline std::vector<LibSVMPoint> 
-importSparseDataReader(
-	std::istream& stream
-) {	
-	std::vector<LibSVMPoint>  fileContents;
-	while(stream){
+importSparseDataReader(std::istream& stream) {
```

Comment:
for VC++ it might help to put this into a separate compilation unit and add a header entry for it. Spirit takes a lot of ressources during compilation